SuperBuild/External_python-extension-manager-ssl-requirements.cmake

```diff
@@ -34,29 +34,29 @@ if(NOT Slicer_USE_SYSTEM_${proj})
-  #  - cryptography-41.0.1-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
-  #  - cryptography-41.0.1-cp37-abi3-manylinux_2_28_aarch64.whl
-  #  - cryptography-41.0.1-cp37-abi3-manylinux_2_28_x86_64.whl
-  #  - cryptography-41.0.1-cp37-abi3-musllinux_1_1_aarch64.whl
-  #  - cryptography-41.0.1-cp37-abi3-musllinux_1_1_x86_64.whl
-  #  - cryptography-41.0.1-cp37-abi3-win_amd64.whl
-  cryptography==41.0.1 --hash=sha256:f73bff05db2a3e5974a6fd248af2566134d8981fd7ab012e5dd4ddb1d9a70699 \
-                       --hash=sha256:1a5472d40c8f8e91ff7a3d8ac6dfa363d8e3138b961529c996f3e2df0c7a411a \
-                       --hash=sha256:7fa01527046ca5facdf973eef2535a27fec4cb651e4daec4d043ef63f6ecd4ca \
-                       --hash=sha256:b46e37db3cc267b4dea1f56da7346c9727e1209aa98487179ee8ebed09d21e43 \
-                       --hash=sha256:d198820aba55660b4d74f7b5fd1f17db3aa5eb3e6893b0a41b75e84e4f9e0e4b \
-                       --hash=sha256:948224d76c4b6457349d47c0c98657557f429b4e93057cf5a2f71d603e2fc3a3 \
-                       --hash=sha256:059e348f9a3c1950937e1b5d7ba1f8e968508ab181e75fc32b879452f08356db \
-                       --hash=sha256:b4ceb5324b998ce2003bc17d519080b4ec8d5b7b70794cbd2836101406a9be31 \
-                       --hash=sha256:1fee5aacc7367487b4e22484d3c7e547992ed726d14864ee33c0176ae43b0d7c
+  #  - cryptography-41.0.4-cp37-abi3-macosx_10_12_universal2.whl
+  #  - cryptography-41.0.4-cp37-abi3-macosx_10_12_x86_64.whl
+  #  - cryptography-41.0.4-cp37-abi3-manylinux_2_17_aarch64.manylinux2014_aarch64.whl
+  #  - cryptography-41.0.4-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+  #  - cryptography-41.0.4-cp37-abi3-manylinux_2_28_aarch64.whl
+  #  - cryptography-41.0.4-cp37-abi3-manylinux_2_28_x86_64.whl
+  #  - cryptography-41.0.4-cp37-abi3-musllinux_1_1_aarch64.whl
+  #  - cryptography-41.0.4-cp37-abi3-musllinux_1_1_x86_64.whl
+  #  - cryptography-41.0.4-cp37-abi3-win_amd64.whl
+  cryptography==41.0.4 --hash=sha256:80907d3faa55dc5434a16579952ac6da800935cd98d14dbd62f6f042c7f5e839 \
+                       --hash=sha256:35c00f637cd0b9d5b6c6bd11b6c3359194a8eba9c46d4e875a3660e3b400005f \
+                       --hash=sha256:cecfefa17042941f94ab54f769c8ce0fe14beff2694e9ac684176a2535bf9714 \
+                       --hash=sha256:e40211b4923ba5a6dc9769eab704bdb3fbb58d56c5b336d30996c24fcf12aadb \
+                       --hash=sha256:23a25c09dfd0d9f28da2352503b23e086f8e78096b9fd585d1d14eca01613e13 \
+                       --hash=sha256:2ed09183922d66c4ec5fdaa59b4d14e105c084dd0febd27452de8f6f74704143 \
+                       --hash=sha256:5a0f09cefded00e648a127048119f77bc2b2ec61e736660b5789e638f43cc397 \
+                       --hash=sha256:9eeb77214afae972a00dee47382d2591abe77bdae166bda672fb1e24702a3860 \
+                       --hash=sha256:c880eba5175f4307129784eca96f4e70b88e57aa3f680aeba3bab0e980b0f37d
   # [/cryptography]
   # Specifying "[crypto]" is required since PyGithub 1.58.1
```

Comment:
Does this comment still apply ?

Reply:
Based on the below requirements specifying the same it appears the change of `PyJWT[crypto]==2.7.0` to `PyJWT==2.8.0` actually needs to be `PyJWT[crypto]==2.8.0` based on what was added in https://github.com/Slicer/Slicer/commit/ac30c6e28118f871c93a153beaf25db1e97f424f?

https://github.com/PyGithub/PyGithub/blob/e584a90f6706c3b590a20e1c9caac7874ba6b122/requirements.txt#L4

https://github.com/PyGithub/PyGithub/blob/14ae2cad4ceeefe9a0f70916af03f0a2017ca213/requirements.txt#L3

Reply:
The drop of `[crypto]` was done by the `python_package_updater.py` script. I have pushed an update to add it back in. Note that I ran a build with a clean portion of python related superbuild packages, but not a completely clean Slicer superbuild.